src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    // Add method
+    add: function (num) {
+      this.result += num;
+    },
+
+    // Subtract method
+    subtract: function (num) {
+      this.result -= num;
+    },
+
+    // Multiply method
+    multiply: function (num) {
+      this.result *= num;
+    },
+
+    // Divide method
+    divide: function (num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+        return 'Cannot divide by zero.';
```

Comment:
Returning a string 'Cannot divide by zero.' might not be the best approach for error handling. Consider throwing an error or handling it in a way that is consistent with the rest of your application logic.